<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="outputText">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="genKeysButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>GenKeys</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="publicKeyText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="publicKeyLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Public Key</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="privateKeyText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="privateKeyLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Private Key</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="encryptRadio">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="decryptRadio">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="proceedButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lockKeysButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock keys</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="signRadio">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Message</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="checkSignRadio">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Message sign</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportKeysButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Keys</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyOutputButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy output to Clipboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="desButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DES</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
